src/components/tag/SearchTag.jsx

```diff
@@ -0,0 +1,61 @@
+import React, { Component } from 'react';
+
+/**
+ * @class CreateArticlePage
+ * @extends {Component}
+ * @param {event}
+ */
+export default class SearchTag extends Component {
+    state = {
+      tags: [],
+    }
+
+  handleEnterKey = (event) => {
+    const { tags } = this.state;
+    const tag = event.target.value;
+    if (tags.length <= 4) {
+      if (event.keyCode === 13) {
+        const currentTags = [...tags, tag];
+        this.setState({
+          tags: currentTags
+        });
+        event.target.value = '';
```

Comment:
#### Critical
Why are you reassigning the value of `event.target.value` again?

Reply:
we are using it to set back the input field to empty